spec/features/idv/analytics_spec.rb

```diff
@@ -153,7 +153,7 @@
                                  vendor_name: 'ResolutionMock',
                                  vendor_workflow: nil,
                                  verified_attributes: nil },
-          state_id: state_id_resolution_with_id_type,
+          state_id: state_id_resolution,
```

Comment:
This is the `let(:in_person_path_proofing_results)` block. As discussed above, we don't set this for IPP now.